crypto/fipsmodule/bn/exponentiation.c

```diff
@@ -119,6 +119,56 @@
 #include "internal.h"
 #include "rsaz_exp.h"
 
+#if !defined(OPENSSL_NO_ASM) &&                                                \
+    (defined(OPENSSL_LINUX) || defined(OPENSSL_APPLE)) &&                      \
+    defined(OPENSSL_AARCH64) && defined(OPENSSL_BN_ASM_MONT)
```

Comment:
why use `defined(OPENSSL_BN_ASM_MONT)`?

Reply:
is this not independent of that?

Reply:
That is a good point - I will make an update